www/include/reporting/dashboard/ajaxReporting_js.php

```diff
@@ -41,6 +41,9 @@
     $arg = "id=" . $id;
 }
 
+$startDate = 0;
+$endDate = 0;
```

Comment:
aren't they already set from the script calling this one ?

Reply:
not every time, but I moved those cause it was overrinding post values